quickstack/manifests/neutron/compute.pp

```diff
@@ -285,4 +285,9 @@
     opt_misc => ['backspace=2','tabstop=4','shiftwidth=4','expandtab','nocp','relativenumber','number','ruler','hlsearch','showcmd','showmatch','ignorecase','smartcase','incsearch','autowrite','hidden']
   }
 
+  # Security Rules fix 
```

Comment:
Can you put "for openstack vm firewalls" at the end of the comment? think it would be helpful to be a bit more clear on what security rules we're changing sysctl stuff for...